packages/node_modules/@webex/plugin-messages/test/integration/spec/messages.js

```diff
@@ -377,6 +377,79 @@ describe('plugin-messages', function () {
           });
       });
     });
+
+    describe('#threadedMessageTests()', () => {
```

Comment:
This describe should go under the `#list()` describe block and be utilized as a "when" case for reading out the test spec:

```suggestion
    describe('when a message is threaded', () => {
```

So the full spec for the first test would read: "plugin-messages #messages #list() when a message is threaded returns all messages for a room". 

(Now, after reviewing the full file, the `#messages` describe block seems to be extraneous.)